src/Catalog/CommonUtility.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public static class CommonUtility
+    {
+        public static Uri GetNugetCacheBustingUri(Uri originalUri)
+        {
+            return GetNugetCacheBustingUri(originalUri, DateTime.Now.ToString());
+        }
+
+        // Adding the timestamp to the URI as a query string ensures
```

Comment:
It was more of a behavioral comment than a functional, but I might as well add it.